plugins/media/wowza/lib/model/WowzaMediaServerNode.php

```diff
@@ -134,8 +134,8 @@ public function getApplicationPrefix($mediaServerConfig)
 
 		$appPrefix = $this->getValueByField($mediaServerConfig, 'appPrefix', $appPrefix);
 		
-		if($this->getAppPrefix())
-			$appPrefix = $this->getApplicationPrefix();
+		if(is_null($this->getAppPrefix()))
+			$appPrefix = $this->getAppPrefix();
```

Comment:
You are right my bad... The not somehow disappeared... Will add it and update the pull request.

Thanks for the heads up.